tools/pylib/_boutcore_build/boutcore.pyx.in

```diff
@@ -857,7 +1129,23 @@ cdef class Options:
         return self
 
     def get(self,name,default):
-        """get(self,name,default)"""
+        """
+        Get an option
+
+        Parameters
+        ----------
+        name : string
+             the name of the value to get. Can be relative,
+             e.g. \`mesh:ddx:first\`.
+        default : various
```

Comment:
This should just be `bool, string or float` if they are the only supported types